analysis/topEFT/topeft.py

```diff
@@ -15,7 +15,7 @@
 
 from topcoffea.modules.GetValuesFromJsons import get_param
 from topcoffea.modules.objects import *
-from topcoffea.modules.corrections import SFevaluator, GetBTagSF, jet_factory, GetBtagEff, AttachMuonSF, AttachElectronSF, AttachPerLeptonFR, GetPUSF
+from topcoffea.modules.corrections import SFevaluator, GetBTagSF, jet_factory, GetBtagEff, AttachMuonSF, AttachElectronSF, AttachPerLeptonFR, GetPUSF, apply_roccor
```

Comment:
We should stick to a convention, something like `ApplyRochesterCorrections` (I guess `jet_factory` slipped passed us!), or even put the corrections inside `SFevaluator`.